frontend/src/pages/mentoringCreate/components/CertificateSection/CertificateSection.tsx

```diff
@@ -0,0 +1,92 @@
+import { useState } from 'react';
+
+import styled from '@emotion/styled';
+
+import CertificateInput from '../CertificateInput/CertificateInput';
+import TitleSeparator from '../TitleSeparator/TitleSeparator';
+
+function CertificateSection() {
+  const [certificates, setCertificates] = useState<number[]>([]);
+  const handleAddButtonClick = () => {
+    setCertificates((prev) => [...prev, Date.now()]);
+  };
+
+  const handleDeleteButtonClick = (id: number) => {
+    setCertificates((prev) => prev.filter((item) => item !== id));
+  };
```

Comment:
p4: crypto.randomUUID()를 사용하는 건 어떨까? 🤔 

Date.now()는 짧은 시간 동안 여러 번 호출하면 같은 밀리초 값을 반환할 수 있어서, 리스트 key로 사용하면 충돌이 발생할 수 있을 거 같아!

[crypto.randomUUID()](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID)도 Date처럼 자바스크립트에 내장된 기능인데, 매우 고유한 UUID를 생성해서 안전할 거 같아 😊

```suggestion
const [certificates, setCertificates] = useState<string[]>([]);
const handleAddButtonClick = () => {
    setCertificates((prev) => [...prev, crypto.randomUUID()]);
};

const handleDeleteButtonClick = (id: string) => {
    setCertificates((prev) => prev.filter((item) => item !== id));
};
```

Reply:
안그래도 날짜값으로 키값 지정해주는데 있어서 오류가 있지 않을까 했는데 역시 문제가 있었구나.. 
골치아팠는데 crypto.randomUUID() 라는 좋은 방법 배워갑니당~